packages/headless/src/database/mongodb/customer/index.ts

```diff
@@ -1,38 +1,62 @@
 import { Injectable } from '@nestjs/common';
 import { Customer, CustomerAddress } from 'src/entity/customer';
+import { Otp } from 'src/entity/otp';
 import { ICustomerDatabase } from 'src/modules/customer/repositories/customer.database.interface';
+import { OtpModel } from '../otp/otp.model';
 import { CustomerModel } from './customer.model';
+const ONE_HOUR = 60 * 60 * 1000;
 
 @Injectable()
 export class CustomerDatabase implements ICustomerDatabase {
+
+  async sendOtp(data: Otp): Promise<Otp | null> {
+    return await OtpModel.create(data);
+  }
+
+  async verifyOtp(query: Record<string, any>): Promise<Otp | null> {
+    const otp = await OtpModel.findOne(query);
+    if (otp) await this.updateOtp(query, { otpVerifiedAt: Date.now() + ONE_HOUR, isVerified: true });
```

Comment:
keep otpVerifiedAt: Date.now(). so that you can write any logic based on that timestamp in service layer. and you can change this validation time instantly. otherwise if i say to change this validation time to 10 minutes, you will have to update some data from database.